.github/scripts/run-linux-poweroff-vitis.py

```diff
@@ -30,10 +30,7 @@ def run_linux_poweroff_vitis():
 
                 run("./marshal -v install test/outputs.yaml")
 
-            # download prebuilt xclbin to /tmp
-            # this CI image is managed by the default CI account
-            with prefix('cd /tmp'):
-                run('wget https://firesim-ci-vitis-xclbins.s3.us-west-2.amazonaws.com/firesim_rocket_singlecore_no_nic_d148b73.xclbin')
+            # No download, edit config_hwdb.yaml directly and specify URI there
```

Comment:
```suggestion
```

Nit: It's preferable to make this as a comment in the PR (to help with review) because the comment (without any accompanying code) will lose context over time.